core/modules/system/config/system.core.json

```diff
@@ -42,6 +42,7 @@
     "image_toolkit": "gd",
     "image_jpeg_quality": "75",
     "image_style_flood_limit": 5,
+    "initial_version": "",
```

Comment:
Calling this "install_version" I think would line it up with "install_profile".

Reply:
I had a reason for changing it...  my variables are all still named `$install_version`, and I know I had used that name *initially*. But I don't remember now why I changed it :/

I'm open to putting it back :)